setup.cfg

```
@@ -20,6 +20,7 @@ packages = find:
 python_requires = >=3.7
 install_requires =
     packaging
+    'Jinja2<3.1;sphinx<4'
     sphinx>=1.7
```

Maybe something like that ?
```suggestion
    Jinja2<3.1
    sphinx>=1.7,<4
```

---

What I really want to do is... "if Sphinx is less than 4, then also pin maxversion of Jinja2, otherwise use the latest Jinja2 no problem"

---

Hum, not sure if you can do that ...

---

Hmm... Is it too controversial to bump minversion of sphinx because of this? Should I just pin jinja2 in the affected tox env?